plugins/dice.js

```diff
@@ -36,12 +56,23 @@ module.exports = (function(){
 		let aDiceRolls = message.text.match(reDice),
 			aResults = [],
 			returnMessage = bot.makeMention(user) + ': You rolled:';
-
 		for (let element of aDiceRolls) {
-			let [die, operator, modifier] = element.split(/(\+|-|\*|\/)/g),
+			let [diePart, quantifier, target] = element.split(/(&[g|G|l|L]t;)/g),
+				[die, operator, modifier] = diePart.split(/(\+|-|\*|\/)/g),
 				roll = rollDice(die);
+			console.log({
+				diepart: diePart,
+				quantifier: quantifier,
+				target: target,
+				die: die,
+				operator: operator,
+				modifier: modifier
+			});
 			if (operator) roll.result = diceCalculation(roll.result, operator, modifier);
-			returnMessage = returnMessage + ' ' + element + ' = ' + roll.result + ' [' + roll.rolls.toString() + ']';
+			if (quantifier && target) roll.target = checkTarget(roll.result, quantifier, target);
+			roll.target = roll.target || '';
```

Comment:
if you swapped 72 and 73 youd get rid of the ||, just set to blank and if quantifier and target overwrite it.